CHANGELOG.rst

```diff
@@ -4,6 +4,10 @@ Changelog
 in development
 --------------
 
+0.9.0
+-----
+* Update dependencies (improvement)
+* Upgrade from hubot 2 to 3 (improvement)
```

Comment:
I think it's more fits the `change` category, rather then improvement.

Also worth including in Changelog other goodies like deprecation of older nodejs versions as a `change` and adding new nodejs versions as supported.